docs/guides/cluster-to-advanced-cluster-migration-guide.md

```diff
@@ -0,0 +1,181 @@
+# Cluster to Advanced Cluster Migration Guide
+
+**Objective**: Guide users to replace the `mongodbatlas_cluster` resource with the `mongodbatlas_advanced_cluster` resource.
+
+**Note**: This guide focus on the resource migration as the data source migration is only requiring a resource_type change from `data.mongodbatlas_cluster` to `data.mongodbatlas_advanced_cluster`.  However, pay attention to the [output changes.](#output-changes)
```

Comment:
knit: in case you want to have color boxes, see: https://developer.hashicorp.com/terraform/registry/providers/docs#callouts

Reply:
Thanks for the tip!
Do you think it will help here?
Do we have any guidance on when to use/not use colors?

Reply:
try out: `guides/advanced-cluster-new-sharding-schema#migration-sharded`